build-logic/testing-convention/src/main/kotlin/convention.unit-testing.gradle.kts

```diff
@@ -56,8 +54,11 @@ plugins.withType<KotlinBasePluginWrapper>() {
         add("testRuntimeOnly", libs.junitPlatformRunner)
         add("testRuntimeOnly", libs.junitPlatformLauncher)
 
-        if (onlyInSubprojects() && project.name != "truth-extensions") {
-            add("testImplementation", project(":subprojects:common:truth-extensions"))
+        if (onlyInSubprojects()) {
+            if (project.name != "truth-extensions") {
+                add("testImplementation", project(":subprojects:common:truth-extensions"))
+            }
+            add("testImplementation", testFixtures(project(":subprojects:common:logger")))
```

Comment:
I seriosly doubt adding implicit dependencies.
As for me, they are widely used but still optional.

I'm ok with adding them explicitly. 
I see `convention.unit-testin` as junit specific thing.

Reply:
agree, removed it